### PR TITLE
OCSADV-354B: Change string and add tooltip for undefined user SEDs.

### PR DESCRIPTION
**Pull request by fnussber**
Changes to a string, addition of a tooltip and a small refactoring.

### CODE REVIEW

File: bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala [hunk 1]
```diff
@@ -28,25 +27,33 @@ class ItcServiceImpl extends ItcService {
...
-      val spectrum      = new String(spectrumBytes)
-      UserDefinedSpectrum(name, spectrum)
+    // update parameters sent from client with stuff that needs to be done on the server
+    val updatedParams: ItcParameters = {
+
+      // Get the SED data from an aux file. For now we can assume that the ITC service is running on the same
+      // machine as the database (localhost). In case this setup changes, we need to change this here, too.
+      def readAuxFile(id: String, name: String): SpectralDistribution = {
+        val programId     = SPProgramID.toProgramID(id)
+        val spectrumBytes = new AuxFileClient("localhost", 8443).fetchToMemory(programId, name)
```
Comment (cquiroz) [inline bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala]: This looks like a future source of bugs :smile:, but I don't know how you could make it configurable in a sensible way

**Comment by cquiroz** [comment]
the code looks fine :+1:

File: bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala (hunk 1 above)
Comment (fnussber) [inline bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala]: You mean localhost and port number? Yeah, that's a point. But making it configurable only helps if people remember to change the configuration.. pick your poison, I guess?

Comment (cquiroz) [inline bundle/edu.gemini.itc/src/main/scala/edu/gemini/itc/service/ItcServiceImpl.scala]: That's right, we'll likely forget either one

**Pull request merged**